app/controllers/pages_controller.rb

```diff
@@ -0,0 +1,7 @@
+class PagesController < ApplicationController
+
+	def home
+		@blogs = Blog.paginate(:page => params[:page], :per_page => 5)
```

Comment:
Do not display draft blogs and order it by creation date (newly created on top) @jedilaunchlabs